OptimizelySDK.Net35/OptimizelySDK.Net35.csproj

```diff
@@ -120,6 +120,9 @@
     <Compile Include="..\OptimizelySDK\Logger\NoOpLogger.cs">
       <Link>Logger\NoOpLogger.cs</Link>
     </Compile>
+	<Compile Include="..\OptimizelySDK\Notifications\NotificationCenter.cs">
```

Comment:
it's still showing old commits from master, need to recheck it.

Reply:
Yes I saw that. Might be the result of excessive branch merging. I will fix it when sending pull request to Optimizely.